README.md

```diff
@@ -12,3 +12,83 @@ Playbook for deploying OSBS into [OpenShift Dedicated](https://www.openshift.com
 
 4. Run `deploy.sh`. Any arguments will be passed to `ansible-playbook` - you
    might want to use e.g. `--verbose`.
+
+
+Requirements for self-hosted OSE instance
+=========================================
+
+A bare self-hosted OSE installation should be sufficient for OSBS to work,
+as it doesn't require routers, docker registry or SDN to function.
+
+
+Role Variables
+==============
+
+Specify project namespace
+```
+osbs_namespace: best-namespace
+```
+
+Create service accounts. 
+
+Depending on your configuration OSBS needs 
+service accounts to communicate with koji and pulp (see osbs-client configuration).
+```
+osbs_service_accounts:
+  - koji
+```
+
+Set permissions for users
+```
+osbs_readonly_users: []
+osbs_readonly_groups: []
+osbs_readwrite_users:
+  - system:serviceaccount:{{ osbs_namespace }}:koji
+  - system:serviceaccount:{{ osbs_namespace }}:builder
+osbs_readwrite_groups:
+  - system:authenticated
+osbs_admin_users: []
+osbs_admin_groups: []
+```
+
+Set maximum number of simultanious builds
+```
+osbs_master_max_pods: -1
+```
+
+Skip errors on importing secrets, if those are not required
+```
+osbs_secret_can_fail: true
+```
+
+Read pulp credentials from secret and put them in a local dir
```

Comment:
It's the other way around I think: the credentials are read from the local directory and used to create the secret.